Classes/Domain/AbstractEventSourcedAggregateRoot.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Neos\Cqrs\Domain;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cqrs\Event\EventTransport;
+use Neos\Cqrs\EventStore\EventStream;
+use Neos\Cqrs\RuntimeException;
+
+/**
+ * AggregateRootTrait
```

Comment:
Oops, just copy/pasted this. But I found some more typos and would like to address those in a separate PR